Makefile

```
@@ -0,0 +1,9 @@
+TOPTARGETS := all clean
+
+SUBDIRS := go
+
+$(TOPTARGETS): $(SUBDIRS)
+$(SUBDIRS):
+	$(MAKE) -C $@ $(MAKECMDGOALS)
+
+.PHONY: $(TOPTARGETS) $(SUBDIRS)
```

No newline at end of file
```suggestion
.PHONY: $(TOPTARGETS) $(SUBDIRS)

```